### DISCUSSION TITLE
Use for traditional lists in addition to grid?

### DISCUSSION THREAD

**Discussion by ifandelse** [rocwang/vue-virtual-scroll-grid#78] [Q&A] [answered]
Hi there! I have an app that will have an instance of a grid list as well as several plain "1 item per row" lists. I've just discovered this lib, and was curious if you'd recommend it for both scenarios (tackling the latter by using a 1 column grid, etc.), or if you think it's best to focus on this lib for the grid and use a different for typical virtualized lists? Thanks!

**Comment by rocwang** [comment, marked as answer]
Yes, I think using a "1 column grid" to render your items as a list is totally fine. This could potentially help you avoid introducing another "list" library.
